tests/installation/handle_reboot.pm

```diff
@@ -0,0 +1,25 @@
+use base "opensusebasetest";
```

Comment:
Please, add header here with the summary it does.

Reply:
Done.